lib/apexcharts/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ApexCharts
-  VERSION = '0.1.6'
+  VERSION = '0.1.7'
```

Comment:
No need to update the version just yet. I'm assigning myself as the release manager. So, let me decide when to bump up the version. Usually, I bump the version up after adding a few features and tested the them on my app a few times.

Reply:
No worries I've removed that.